pages/common/nano.md

```diff
@@ -18,3 +18,7 @@
 - Indent new lines to the previous lines' indentation:
 
 `nano -i {{filename}}`
+
+- Before saving, back up the previous version of the file with name = "currentName+tilde" (useful for quick edits)
```

Comment:
Maybe this is better:
```suggestion
- Back up the previous version as `{{filename}}~` when save:
```
or
```suggestion
- Before saving, back up the previous version as `{{filename}}~`:
```

Reply:
Maybe `on save` instead of `when save` there @navarroaxel? Other than that, either is fine :+1:

Reply:
If we're going that way, how about this?:
```suggestion
- Before saving, back up the previous version of the file with name = "currentName+tilde" (useful for quick edits)
+ Before modification, backup separately as `{{currentFileName}}~`
```

Reply:
@sbrl ?

Reply:
That would work too! Let's do that. Although, we prefer `snake_case` to `pascalCase`:
```suggestion
- Before modification, backup separately as `{{current_file_name}}~`:
```